packages/eas-cli/src/build/android/build.ts

```diff
@@ -113,14 +115,22 @@ async function ensureAndroidCredentialsAsync(
   if (!shouldProvideCredentials(ctx)) {
     return;
   }
+  const androidApplicationIdentifier = await getOrConfigureApplicationIdAsync(
```

Comment:
```suggestion
  const androidApplicationIdentifier = getApplicationId(
```
This change is related to https://github.com/expo/eas-cli/pull/431#discussion_r641394021
The application id must be already configured in this context.

Reply:
Sorry for making you change it again but in https://github.com/expo/eas-cli/pull/427/files/887877aaf9874f6b77a09f3c8632456d54f98355#r640438388 I meant to use the result of the call on line 70. 
However, when I was reviewing https://github.com/expo/eas-cli/pull/431 I realized that we should change the name of  `getOrConfigureApplicationIdAsync` 😅